content/auth/capabilities.textile

```diff
@@ -145,15 +142,15 @@ If a set of capabilities are requested, and the intersection between those and t
 Using the following example, an API key exists with the listed capabilities. If an "Ably Token":/auth/token#tokens is requested that specifies a set of capabilities, and there is no intersection between the capabilities of the issuing API key and requested token, then the token request will be rejected. In the following example, the callback will be returned with an error.
 
 ```[javascript]
-// API key capabilities:
-{
-  "chat": ["*"]
-}
-
-// Token request that specifies capabilities:
-auth.requestToken({ capability: {
-  "status": ["*"]
-}}, tokenCallback)
+  // API key capabilities:
+  {
+    'chat': ['*']
+  }
+
+  // Token request that specifies capabilities:
+  const requestToken = auth.requestToken({ capability: {
```

Comment:
```suggestion
  const tokenDetails = await auth.requestToken({ capability: {
```

Reply:
be careful here - `requestToken` is an async method and returns a `TokenDetails` object (as opposed to `createTokenRequest` which returns a `TokenRequest`)

Reply:
https://github.com/ably/docs/pull/2119/commits/536a260694f5faa322f0122cf78af5946118313a

Updated. Good point.